base/applications/taskmgr/lang/en-US.rc

```diff
@@ -239,38 +238,38 @@ BEGIN
-    CONTROL "CPU 5", IDC_CPU5, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 11, 93, 37, 10
-    CONTROL "CPU 6", IDC_CPU6, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 11, 106, 37, 10
-    CONTROL "CPU 7", IDC_CPU7, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 11, 119, 37, 10
-    CONTROL "CPU 8", IDC_CPU8, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 28, 37, 10
-    CONTROL "CPU 9", IDC_CPU9, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 41, 37, 10
-    CONTROL "CPU 10", IDC_CPU10, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 54, 41, 10
-    CONTROL "CPU 11", IDC_CPU11, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 67, 41, 10
-    CONTROL "CPU 12", IDC_CPU12, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 80, 41, 10
-    CONTROL "CPU 13", IDC_CPU13, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 93, 41, 10
-    CONTROL "CPU 14", IDC_CPU14, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 106, 41, 10
-    CONTROL "CPU 15", IDC_CPU15, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 61, 119, 41, 10
-    CONTROL "CPU 16", IDC_CPU16, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 28, 41, 10
-    CONTROL "CPU 17", IDC_CPU17, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 41, 41, 10
-    CONTROL "CPU 18", IDC_CPU18, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 54, 41, 10
-    CONTROL "CPU 19", IDC_CPU19, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 67, 41, 10
-    CONTROL "CPU 20", IDC_CPU20, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 80, 41, 10
-    CONTROL "CPU 21", IDC_CPU21, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 93, 41, 10
-    CONTROL "CPU 22", IDC_CPU22, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 106, 41, 10
-    CONTROL "CPU 23", IDC_CPU23, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 116, 119, 41, 10
-    CONTROL "CPU 24", IDC_CPU24, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 28, 41, 10
-    CONTROL "CPU 25", IDC_CPU25, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 41, 41, 10
-    CONTROL "CPU 26", IDC_CPU26, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 54, 41, 10
-    CONTROL "CPU 27", IDC_CPU27, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 67, 41, 10
-    CONTROL "CPU 28", IDC_CPU28, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 80, 41, 10
-    CONTROL "CPU 29", IDC_CPU29, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 93, 41, 10
-    CONTROL "CPU 30", IDC_CPU30, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 106, 41, 10
-    CONTROL "CPU 31", IDC_CPU31, "Button", BS_AUTOCHECKBOX | WS_DISABLED | WS_TABSTOP, 175, 119, 41, 10
+    AUTOCHECKBOX "CPU 0", IDC_CPU0, 11, 28, 37, 10
+    AUTOCHECKBOX "CPU 1", IDC_CPU1, 11, 41, 37, 10, WS_DISABLED | WS_TABSTOP
```

Comment:
Have you verified, e.g. with resource hacker, whether one needs to explicitly add the WS_TABSTOP style?

Reply:
yes, the WS_TABSTOP remains for those who do not have any explicit other style set. But for those where we want to have the WS_DISABLED, like here in this line we do have to do the explicit WS_TABSTOP also. Resolved.